hw_diag/tests/diagnostics/test_diagnostic_report.py

```diff
@@ -0,0 +1,46 @@
+import unittest
```

Comment:
DiagnosticsReport was moved to hm_pythelper along with this [test](https://github.com/NebraLtd/hm-pyhelper/blob/master/hm_pyhelper/tests/test_diagnostic_report.py).